spec/features/choices_spec.rb

```diff
@@ -0,0 +1,20 @@
+# the marketeer will be presented the choices (rock, paper and scissors)
+
+feature 'Present the choices' do
+  scenario 'it presents the choices (rock, paper and scissors)' do
+    visit '/'
```

Comment:
Maybe create a 'sign in method' that automatically visits the homepage and fills in the players name, then call the method inside tests so you won't have to keep on repeating lines 5-7 in the feature tests